201-api-management-create-with-multiregion/azuredeploy.parameters.json

```diff
@@ -8,11 +8,8 @@
         "publisherName": {
             "value": "templateTest"
         },
-        "additionalLocation1": {
-            "value": ""
-        },
-        "additionalLocation2": {
-            "value": ""
+        "additionalLocations": {
+            "value": ["NorthCentralUS", "East US2"]
```

Comment:
remove this and use the defaultValue for CI

Reply:
can you point me to the variable to use here? I could not find any [here](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#parameters-file-placeholders) 
The requirement here is that `additionLocations` **must** be different from the `resourceGroup` location.

Reply:
CI will deploy into westus - you already have a defaultValue just use that and remove this from the param file